packages/console/src/scss/table.module.scss

```diff
@@ -31,3 +31,7 @@ tr.clickable {
     }
   }
 }
+
+.empty {
+  height: 100%;
```

Comment:
maybe 50% is enough? Idk.

Reply:
100% looks better, IMO. I'm sticking to it 😎